docs/articles/vs-test-adapter/Tips-And-Tricks.md

```diff
@@ -181,13 +181,22 @@ However, it has been seen to also have adverse effects, so use with caution.
 
 #### ConsoleOut
 
-When set to 1, default, will send Console standard output to the Visual Studio Output/Test window, and also with dotnet test, it will appear here. (Note: You have to use the '-v n' option)
+When set to 1 or 2, 2 is default, will send Console standard output to the Visual Studio Output/Test window, and also with dotnet test, it will appear in the output.
```

Comment:
```suggestion
When set to 1 or 2 (2 is default), will send Console standard output to the Visual Studio Output/Test window, and also with dotnet test, it will appear in the output.
```

Minor nitpick